targets/TARGET_RDA/TARGET_UNO_91H/PinNames.h

```diff
@@ -0,0 +1,148 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2013 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef MBED_PINNAMES_H
+#define MBED_PINNAMES_H
+
+#include "cmsis.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef enum {
+    PIN_INPUT,
+    PIN_OUTPUT
+} PinDirection;
+
+#define PORT_SHIFT  8
+
+typedef enum {
+    PA_0  = (0 << PORT_SHIFT | 0 ),
+    PA_1  = (0 << PORT_SHIFT | 1 ),
+    PA_2  = (0 << PORT_SHIFT | 2 ),
+    PA_3  = (0 << PORT_SHIFT | 3 ),
+    PA_4  = (0 << PORT_SHIFT | 4 ),
+    PA_5  = (0 << PORT_SHIFT | 5 ),
+    PA_6  = (0 << PORT_SHIFT | 6 ),
+    PA_7  = (0 << PORT_SHIFT | 7 ),
+    PA_8  = (0 << PORT_SHIFT | 8 ),
+    PA_9  = (0 << PORT_SHIFT | 9 ),
+    PB_0  = (1 << PORT_SHIFT | 0 ),
+    PB_1  = (1 << PORT_SHIFT | 1 ),
+    PB_2  = (1 << PORT_SHIFT | 2 ),
+    PB_3  = (1 << PORT_SHIFT | 3 ),
+    PB_4  = (1 << PORT_SHIFT | 4 ),
+    PB_5  = (1 << PORT_SHIFT | 5 ),
+    PB_6  = (1 << PORT_SHIFT | 6 ),
+    PB_7  = (1 << PORT_SHIFT | 7 ),
+    PB_8  = (1 << PORT_SHIFT | 8 ),
+    PB_9  = (1 << PORT_SHIFT | 9 ),
+    PC_0  = (4 << PORT_SHIFT | 0 ),
+    PC_1  = (4 << PORT_SHIFT | 1 ),
+    PC_2  = (4 << PORT_SHIFT | 2 ),
+    PC_3  = (4 << PORT_SHIFT | 3 ),
+    PC_4  = (4 << PORT_SHIFT | 4 ),
+    PC_5  = (4 << PORT_SHIFT | 5 ),
+    PC_6  = (4 << PORT_SHIFT | 6 ),
+    PC_7  = (4 << PORT_SHIFT | 7 ),
+    PC_8  = (4 << PORT_SHIFT | 8 ),
+    PC_9  = (4 << PORT_SHIFT | 9 ),
+    PD_0  = (5 << PORT_SHIFT | 0 ),
+    PD_1  = (5 << PORT_SHIFT | 1 ),
+    PD_2  = (5 << PORT_SHIFT | 2 ),
+    PD_3  = (5 << PORT_SHIFT | 3 ),
+    PD_9  = (5 << PORT_SHIFT | 9 ), // Fake pin for GPADC_VBAT
+
+    UART0_RX = PA_0,
+    UART0_TX = PA_1,
+    UART1_RX = PB_1,
+    UART1_TX = PB_2,
+
+    USBRX   = UART0_RX,
+    USBTX   = UART0_TX,
+
+    I2C_SCL = PC_0,
+    I2C_SDA = PC_1,
+
```

Comment:
no SPI peripheral brought out?

Reply:
1 SPI master supported, Arduino like pin layout provided on the board (D10~D13).

Reply:
In that case, you may want to define the Arduino pin maps also here..thanks!